src/query/service/src/pipelines/processors/transforms/window/transform_window.rs

```diff
@@ -302,18 +302,22 @@ impl<T: Number> TransformWindow<T> {
     /// This function is used for both `ROWS` and `RANGE`.
     fn advance_frame_end_current_row(&mut self) {
         // Every frame must be processed to the end of the input block if the its partition is started.
-        debug_assert!(
```

Comment:
Now you can revert this commit, and add frame in cume_dist test.

ref: 
https://github.com/datafuselabs/databend/issues/16093